api/src/pcapi/core/bookings/models.py

```diff
@@ -56,6 +56,12 @@ class BookingCancellationReasons(enum.Enum):
     REFUSED_BY_INSTITUTE = "REFUSED_BY_INSTITUTE"
     FINANCE_INCIDENT = "FINANCE_INCIDENT"
     BACKOFFICE = "BACKOFFICE"
+    BACKOFFICE_EVENT_CANCELLED = "BACKOFFICE_EVENT_CANCELLED"
+    BACKOFFICE_OVERBOOKING = "BACKOFFICE_OVERBOOKING"
+    BACKOFFICE_BENEFICIARY_REQUEST = "BACKOFFICE_BENEFICIARY_REQUEST"
+    BACKOFFICE_OFFER_MODIFIED = "BACKOFFICE_OFFER_MODIFIED"
+    BACKOFFICE_OFFER_WITH_WRONG_INFORMATIONS = "BACKOFFICE_OFFER_WITH_WRONG_INFORMATIONS"
```

Comment:
_information_ est invariable en anglais :
```suggestion
    BACKOFFICE_OFFER_WITH_WRONG_INFORMATION = "BACKOFFICE_OFFER_WITH_WRONG_INFORMATION"
```

Reply:
vue avec venceslas oui on s'était planté la dessus